src/backend/joanie/core/utils/issuers.py

```diff
@@ -31,3 +32,36 @@ def generate_document(name: str, context: dict) -> bytes:
         font_config=FontConfiguration(),
     )
     return doc_html.write_pdf(stylesheets=[css], zoom=1)
+
+
+def get_contract_definition_placeholder_default_values():
+    """
+    Return a dictionary with default placeholder values for previewing a template
+    of contract definition.
+    """
+    return {
+        "contract": {
+            "body": _("Default Articles template"),
+            "title": _("Contract Definition"),
+        },
```

Comment:
This context will be used as a preview of the contract the learner will have to sign so we should bind the contract definition body and title.